app/screens/login/login.js

```diff
@@ -55,6 +55,11 @@ export default class Login extends PureComponent {
     constructor(props) {
         super(props);
 
+        this.loginRef = React.createRef();
+        this.passwordRef = React.createRef();
+        this.loginId = '';
+        this.password = '';
+
```

Comment:
This is very very funny, @migbot was asking to do it like this in a previous PR cause the potential change on `_lastNativeText` and in RN 62 they actually moved from class components to functional components and hooks, so the private prop is no longer available :P

Reply:
✂️